index.html

```diff
@@ -328,6 +334,25 @@ <h3>Subtitle and caption rendering synchronization</h3>
           and subtitle changes occur at the same time. Many subtitles therefore
           start on the first frame of the shot and end on the last frame."
         </p>
+        <p>
+         The NorDig technical specifications for DVB receivers for the Nordic
+         and Irish markets [[NORDIG]], section 7.3.1, mandates that receivers
+         support TTML in MPEG-2 Transport Streams. The presentation timing
+         precision for subtitles is specified as being within 2 frames.
+        </p>
+        <p>
+          Another important use case is maintaining synchronization of subtitles
+          during program content with fast dialog. The BBC Subtitle Guidelines,
+          section 5.1 says: "Impaired viewers make use of visual cues from the
```

Comment:
I happen to have inside information that section numbers may be removed from the guidelines document in the future, so it would be good to replace the numbers (also above) with the section heading text. That could be raised as a different issue. (or if it's super helpful to be able to reference by number, maybe I could take that back to the folks working on the subtitle guidelines)

Reply:
The numbers aren't so important. Will the section headings and HTML anchors stay the same? I can add links directly to the relevant sections.

Reply:
Let's update this in a future issue. My main feedback is that it would be good if the fragment URL for each section could remain the same, in case of wording changes.